src/styles/stars.scss

```
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("../images/star.svg") center no-repeat;
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4)  {
+background-image: url("../images/star-active.svg");
+}
```

```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4)  {
  background-image: url("../images/star-active.svg");
}
```
indentation - pay attention to code style